src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  let ACTION_REQUIRED = '';
+  
+
+  for (const action of Object.values(actions)) {
+    ACTION_REQUIRED = action['type'];
+
+    switch (ACTION_REQUIRED) {
+      case 'addProperties':
+        Object.assign(state, action['extraData']);
```

Comment:
Instead of accessing object properties with action['extraData'], use dot notation (action.extraData). Ensure that the 'extraData' property is correct as per the task description.